internal/initialize/security.go

```diff
@@ -19,14 +19,10 @@ import (
 	corev1 "k8s.io/api/core/v1"
 )
 
-// RestrictedPodSecurityContext returns a v1.PodSecurityContext with safe defaults.
-// See https://docs.k8s.io/concepts/security/pod-security-standards/
-func RestrictedPodSecurityContext() *corev1.PodSecurityContext {
+// PodSecurityContext returns a v1.PodSecurityContext with some defaults.
+func PodSecurityContext() *corev1.PodSecurityContext {
```

Comment:
Hmmm, yeah, I suppose this SecurityContext is no longer restricting anything... but we still need this separate from `postgres.PodSecurityContext` because of the extra things we do with that PodSecurityContext. (This comment is just me thinking out loud.)